github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits.yaml

```
@@ -0,0 +1,31 @@
+presubmits:
+  kubevirt/managed-tenant-quota:
+  - name: pull-mtq-unit-test
+    skip_branches:
+      - release-v1.1
+    cluster: kubevirt-prow-workloads
```

This would probably more suited to the smaller cluster - 
```suggestion
    cluster: ibm-prow-jobs
```

---

Done